charts/rancher-logging/0.2.5/questions.yml

```diff
@@ -0,0 +1,2 @@
+rancher_min_version: 2.5.10-alpha1
+rancher_max_version: 2.5.99-0
```

Comment:
nice.

Reply:
No, we don't need the eks and gke operator charts. When Rancher 2.6+ starts up, it will uninstall the existing eks/gke-operator charts and install the rancher/charts version.

There are currently no eks/gke-operator charts in system-charts in 2.6

Reply:
Oh right! I forgot that they will be uninstalled. 😅